### PR TITLE
Add German support

### PR DESCRIPTION
**Pull request by HanSolo80**
Added support for German language with self created dictionary since no official one is available online.
Word definitions fetched from dwds.de.

### CODE REVIEW

**Review by kamilmielnik — CHANGES_REQUESTED** [review]
Awesome work, good job :1st_place_medal: 

----

GitHub won't automatically run GitHub Actions for a first time contributor - I had to trigger it myself. There are some code style violations - fortunately most of them should be automatically fixable with `npm run lint:fix` & `npm run prettier:fix` - please [have a look](https://github.com/kamilmielnik/scrabble-solver/runs/5336954466?check_suite_focus=true).

----

I also see that [tests are now failing](https://github.com/kamilmielnik/scrabble-solver/runs/5336954242?check_suite_focus=true) - this must be due to fact that Polish dictionary has expanded and affected the number of valid results.
Not your fault at all, but since it happened now, can you please update [this line](https://github.com/kamilmielnik/scrabble-solver/blob/master/packages/solver/src/Solver.test.ts#L49) to `61` instead of `60`? :pray: [I've created an issue to improve this in the future](https://github.com/kamilmielnik/scrabble-solver/issues/128).

File: packages/word-definitions/src/parse/parseGerman.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import cheerio from 'cheerio';
+
+import { ParseResult } from '../types';
+
+const parseGerman = (html: string): ParseResult => {
```
[the remark this answers is not in the document]
Comment (HanSolo80) [reply packages/word-definitions/src/parse/parseGerman.ts:5]: Ah, this is frustrating. There is no clear pattern they follow when giving word meanings. And duden.de which is the more or less official source is missing a lot of words, especially the conjugations of words cannot be found there.

[the remark this answers is not in the document]
Comment (kamilmielnik) [reply packages/word-definitions/src/parse/parseGerman.ts:5]: No worries, I think it's good enough for the first implementation :+1:

**Review by kamilmielnik — APPROVED** [review]
:+1:

**Pull request merged**